app/presenters/auction_presenter.rb

```diff
@@ -65,6 +65,11 @@ def initialize(auction)
     to: :auction_rules
   )
 
+  delegate(
+    :small_business?,
```

Comment:
how about chaining the methods here rather than adding more delegations? will help in our effort to make law of demeter violations more obvious for future refactors